src/quemb/molbe/mbe.py

```diff
@@ -726,9 +741,23 @@ def optimize(
             be_.optimize(method, J0=J0, trust_region=trust_region)
             self.ebe_tot = self.ebe_hf + be_.Ebe[0]
             # Print the energy components
-            print_energy(
-                be_.Ebe[0], be_.Ebe[1][1], be_.Ebe[1][0] + be_.Ebe[1][2], self.ebe_hf
-            )
+            if self.use_cumulant:
+                print_energy_cumulant(
```

Comment:
Super picky, but maybe for better readability?
```suggestion
                self.ebe_tot = be_.Ebe[0] + self.ebe_hf
                print_energy_cumulant(
```
and remove line 742